tests/conftest.py

```diff
@@ -53,6 +53,48 @@ def pytest_logger_config(logger_config):
     logger_config.set_formatter_class(OCSLogFormatter)
 
 
+@pytest.fixture()
+def supported_configuration():
```

Comment:
for some purposes, like automation development, we'd want to deploy clusters with less than the required minimum.

Reply:
but if the intention is to use it only for running specific test cases, then this should not be a problem

Reply:
This will be now part of only 2 tests and it can be added to additional tests if needed.